sycl/doc/PreprocessorMacros.md

```diff
@@ -31,7 +31,12 @@ This file describes macros that have effect on SYCL compiler and run-time.
 
 - **SYCL_DISABLE_DEPRECATION_WARNINGS**
 
-  Disables all deprecation warnings in SYCL runtime headers, including SYCL 1.2.1 deprecations.
+  Disables all deprecation warnings in SYCL runtime headers, including SYCL
+  1.2.1 deprecations.
+
+-`**SYCL_DISABLE_CPP_VERSION_CHECK_MESSAGE`**
```

Comment:
```suggestion
- **SYCL_DISABLE_CPP_VERSION_CHECK_MESSAGE**
```

Reply:
Can you rename MESSAGE to WARNING please?